Ghana/GHA_extract_microdata_excel_to_csv.ipynb

```diff
@@ -0,0 +1,86 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 0,
+   "metadata": {
+    "application/vnd.databricks.v1+cell": {
```

Comment:
@bhupatiraju I noticed the two newly added notebooks are in .ipynb instead of .py, which makes code review more difficult. Is this a databricks change?

Reply:
Looks like an unfortunate databricks change because I saw the same .ipynb when I was merging my utils change https://github.com/dime-worldbank/mega-indicators/pull/13/files#diff-2d830e7e29bd24a21d73a4cdb4c7bd2fb081a4357173a84b319e3315403e934f